src/OSPSuite.Presentation/Presenters/Charts/SimpleChartPresenter.cs

```diff
@@ -62,6 +62,11 @@ public interface ISimpleChartPresenter : IPresenter<ISimpleChartView>
       ///    Action run when a series point is hot tracked in the chart
       /// </summary>
       Action<int> HotTracked { set; }
+
+      /// <summary>
+      /// Refresh the display after external changes were made to the chart
+      /// </summary>
+      void Refresh();
```

Comment:
because we expose the CurveChart explicitely, I am pretty sure that there is some code in PKsim or Mobi that is modifying the chart directly AFTER it was bound to the display presenter. Those spots will have to call refresh